src/components/Login.tsx

```diff
@@ -8,10 +8,10 @@ export function Login() {
   const sendEmail = (email: string) => {
     const $id = uuidv4();
     try {
-      createMagicLink($id, email);
-
+      createMagicLink($id, email).then(() => {
+        setMessage("Enviado Correctamente");
```

Comment:
If you're using try catch, use async/await. If you prefer .then, use .catch instead of a try/catch